src/updateAttribute.js

```diff
@@ -56,7 +56,13 @@ function setAttribute(node, attrName, attrValue, oldAttrValue, isSvgAttribute) {
 
       // if the event is getting added first time add a listener
     } else if (!oldAttrValue && attrValue) {
-      node.addEventListener(eventName, patchedHandler);
+      let isCaptureEvent = eventName.endWith('Capture');
+      if(isCaptureEvent) {
+        let cleanedEventName = eventName.replace('Capture', '');
```

Comment:
As we are passing attrName for mapping patched handler. We can do this logic inside `getEventName` only.

Reply:
But I would need capture in the `eventName` for `getPatchedEventHandler(node, attrName, attrValue);` to be able to differentiate between `clickcapture` and `click`

right ?

Reply:
Yes, making the update